langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/internal/OpenAiUtils.java

```diff
@@ -312,6 +314,11 @@ public static AiMessage aiMessageFrom(ChatCompletionResponse response) {
                     : AiMessage.from(text, singletonList(toolExecutionRequest));
         }
 
+        String refusal = assistantMessage.refusal();
+        if (isNullOrBlank(text) && isNotNullOrBlank(refusal)) {
```

Comment:
```suggestion
        if (isNotNullOrBlank(refusal)) {
```

I would remove `isNullOrBlank(text)` condition. The `isNotNullOrBlank(refusal)` seems to be enough